src/models.py

```diff
@@ -42,12 +57,7 @@ def check_if_both_age_and_country(self) -> Self:
         return self
 
 
-class SuperUser(BaseModel, validate_assignment=True):
-    user_id: PositiveInt
-    username: Annotated[str, StringConstraints(min_length=2, pattern=r"^[a-zA-Z]*$")]
-    email: EmailStr
-    age: Optional[PositiveInt]
-    country: Optional[CountryAlpha2]
+class SuperUser(User, validate_assignment=True):
     superuser_id: PositiveInt
     role: SuperUserRoles
```

Comment:
Sorry for the confusion about the user-superuser models, I didn't check models in this PR because I thought it was the same as in the previous one. :) 

`SuperUser` is a subclass of class `User`. Hence it inherits all fields from `User`, but it just has some extra features on top (in this case it is just the `role`). So you don't need another ID for the `SuperUser` since it already gets a (unique) id from it being a `User`.

Reply:
Ok, sure! superuser_id deleted :)